code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -175,38 +175,73 @@
 /mob/living/carbon/human/proc/commune()
 	set category = "Abilities"
 	set name = "Commune with creature"
-	set desc = "Send a telepathic message to an unlucky recipient."
+	set desc = "Send a telepathic message to a recipient."
+
+	var/obj/item/organ/external/rhand = src.get_organ("r_hand")
+	var/obj/item/organ/external/lhand = src.get_organ("l_hand")
```

Comment:
This is probably personal preference but a better way to do this is to just use get_active_hand() and check if the hand exists and isn't carrying anything.